docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -4,116 +4,161 @@ title: Cellular
 
-**Cat-M1**, **NB-IoT**, and **GSM** (Global System for Mobile Communications) are all cellular network technologies with distinct characteristics. **GSM**, the most widely used cellular standard globally, provides higher data rates compared to **NB-IoT** and **Cat-M1** but has higher power consumption. In contrast, **Cat-M1** and **NB-IoT** are optimized for IoT applications. **Cat-M1** offers higher data rates and mobility, while **NB-IoT** provides ultra-low power consumption and extended coverage. The choice among **GSM**, **Cat-M1**, and **NB-IoT** depends on the specific requirements of the application, considering factors such as data rates, power consumption, and coverage area.
+| Modules / Network Modes   | Cat-M1 (LTE-M or eMTC) | NB-IoT | GSM/GRPS 2G | GNSS   |
+|---------------------------|------------------------|--------|-------------|--------|
+| Quectel BG95-M3           | ✔                     | ✔      | ✔           | ✔      |
+| Quectel M95               | -                      | -      | ✔           | -      |
 
+# Hardware configuration
 
-## Network configuration
+## Quectel BG95M3 with NimbeLink Skywire click board
 
-Using Cellular Meadow, you don't need to understand the complexities of AT commands or invest time studying module datasheets to enable the cellular network interface.
+To setup the hardware, you could use a Skywire click adapter, which hosts NimbeLink/Skywire ™ cellular modems (using stacking headers) to MikroElektronika development boards. 
 
-You only need to create a **cell.config.yaml** file, where the cell settings will be defined, define the cell interface on the **meadow.config.yaml**, and reserve some pins used by the cell module.
+![Quectel BG95M3 with NimbeLink Skywire click board](images/modem-skywire-click.jpg)
 
-Then, to configure your network you just need to follow these three steps:
+To configure the hardware, start by connecting the necessary jumpers for communication between the Meadow device and the cell module. Then make the necessary connections to supply and turn on the cell module. Finally, connect an antenna to the click board.
 
-1. Create a **cell.config.yaml** and flash it to the device, here's an example:
+### Using a Meadow F7v2 Feather
 
-    ```yaml
-    Settings:
-        APN: virtueyes.com.br # Access Point Name
-        Module: BG95M3 # Module model (BG770A, BG95M3 or M95)
-        User: virtu # APN user (optional)
-        Password: virtu # APN password (optional)
-        Operator: 72410 # Carrier numeric operator code (optional)
-        Mode: CATM1 # Network mode (CATM1, NBIOT or GSM) (optional)
-        Interface: /dev/ttyS1 # Serial interface (UART1 (COM1) = /dev/ttyS0, UART4 (COM4) = /dev/ttyS1, UART6 = /dev/ttyS3) (optional)
-        TurnOnPin: D10 # Pin used to turn on the Cell module (optional)
-        ScanMode: false # Activate the cell network scanning mode (optional)
-    ```
+<img src="images/meadow-modem.jpg" 
```

Comment:
Feel free to just use the Markdown image syntax for images so they size consistently across the site:

```
![images/meadow-modem.jpg](Whatever alt text describes your image.)
```

We should always have alt text too for any non-trivial images.